src/modules/admin/sections/LinkEditor/EditableLink/EditableLink.tsx

```diff
@@ -34,6 +34,7 @@ export default function EditableLink({
   draggableProps: { provided },
 }: EditableLinkProps) {
   const { id, isVisible, title, url } = link;
+  const [showDelete, setShowDelete] = useState(false);
```

Comment:
FYI. there is a custom `useToggle` hook, it's made to toggle the state between `true/false`.

You can refactor this with that if you want to.

Reply:
Fixed: refactored  it with `framer-motion`'s `useCycle` hook